ui/components/component-library/picker-network/picker-network.stories.js

```diff
@@ -53,4 +57,16 @@ export const Src = (args) => (
   </Box>
 );
 
+export const MaxWidth = (args) => (
+  <Box display={DISPLAY.FLEX} flexDirection={FLEX_DIRECTION.COLUMN} gap={2}>
+    <PickerNetwork {...args} label="Optimism" />
+    <PickerNetwork {...args} label="x" />
+    <PickerNetwork
+      {...args}
+      style={{ maxWidth: '200px' }}
+      label="BNB Smart Chain (previously Binance Smart Chain Mainnet)"
+    />
+  </Box>
+);
+
```

Comment:
I think we can just make a note about setting max-width in the label story and docs so this can be removed.